src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_FixedStatement.cs

```diff
@@ -286,6 +290,106 @@ public override BoundNode VisitFixedLocalCollectionInitializer(BoundFixedLocalCo
             return factory.Block(pinnedTempInit, localInit);
         }
 
+        /// <summary>
+        /// fixed(int* ptr = &amp;v){ ... }    == becomes ===>
+        /// 
+        /// pinned ref int pinnedTemp = ref v;    // pinning managed ref
+        /// int* ptr = (int*)&amp;pinnedTemp;         // unsafe cast to unmanaged ptr
```

Comment:
Nit: use CDATA to avoid escaping ampersand

Reply:
With the CDATA, "&amp;" can be changed to "&" on the first line.

---
In reply to: [167655416](https://github.com/dotnet/roslyn/pull/24469#discussion_r167655416) [](ancestors = 167655416)